enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -176,6 +176,34 @@ func openFile(ctx context.Context, env environment.Env, fileName string) (io.Rea
 	return env.GetFileResolver().Open(fileName)
 }
 
+// Returns the cgroup version available on this machine, which is returned from
+//
+//	stat -fc %T /sys/fs/cgroup/
+//
+// and should be either 'cgroup2fs' (version 2) or 'tmpfs' (version 1)
+//
+// More info here: https://kubernetes.io/docs/concepts/architecture/cgroups/
+// TODO(iain): preemptively get this in a provider a la Provider in podman.go.
+func getCgroupVersion() (string, error) {
+	if *firecrackerCgroupVersion != "" {
+		return *firecrackerCgroupVersion, nil
+	}
+	b, err := exec.Command("stat", "-fc", "%T", "/sys/fs/cgroup/").Output()
+	if err != nil {
+		log.Debugf("Error determining system cgroup version: %s", err.Error())
+		return "", err
+	}
+	v := strings.TrimSpace(string(b))
+	if v == "cgroup2fs" {
+		return "2", nil
+	} else if v == "tmpfs" {
+		return "1", nil
+	} else {
+		log.Debugf("Error determining system cgroup version. System-reported version: %s", v)
```

Comment:
this log statement is redundant since the caller is going to report the error and it has the same information as this log

Reply:
Done